config/config.json

```diff
@@ -0,0 +1,31 @@
+{
+    "spark_app": {
+        "app_name": "prepare_step",
```

Comment:
guess: that this doesn't work; you are seeing a warning that this config is not considered.
The configuration parameter is called `spark.app.name`.
https://spark.apache.org/docs/latest/configuration.html